rllib/agents/dqn/dqn.py

```diff
@@ -66,23 +90,23 @@
         "n_step": 1,
 
         # === Replay buffer ===
-        # Size of the replay buffer. Note that if async_updates is set, then
-        # each worker will have a replay buffer of this size.
+        # Deprecated, use capacity in replay_buffer_config instead.
         "buffer_size": DEPRECATED_VALUE,
-        # Prioritized replay is here since this algo uses the old replay
-        # buffer api
-        "prioritized_replay": True,
```

Comment:
Nit: Can we keep this and set it to `DEPRECATED_VALUE` as well, then warn if it's used by a user (same logic as with deprecated config key: `buffer_size` above)?

Reply:
Hey, I updated this to be in the SimpleQ buffer config, also set to DEPRECATED_VALUE, along with the other buffer related parameters.
Just in case someone puts the parameter in it's old place **anywhere**: Then RLlib doesn't error out while merging dicts with unknown keys but instead takes action and warns in validate_config().

Reply:
I have now added the other prioritization parameters.